ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -174,7 +174,7 @@ tests:
     owner: rh-ecosystem-edge
     product: ocp
     timeout: 1h0m0s
-    version: "4.15"
+    version: "4.16"
```

Comment:
Why are we testing this one with 4.16 instead of 4.17? I don't think we even have a pool for 4.16.

Reply:
you are right, Im trying to isolate the issue because it looks like the problem is not with the busybox image